HDDS-12552. Fix Raw use of parameterized class 'SCMCommand'.

nandakumar131 opened the pull request:
## What changes were proposed in this pull request?
Fix Raw use of parameterized class 'SCMCommand'.



## What is the link to the Apache JIRA
HDDS-12552

## How was this patch tested?
CI : https://github.com/nandakumar131/ozone/actions/runs/13787862336

adoroszlai reviewed (commented):
Thanks @nandakumar131 for the patch.

Found one more:

https://github.com/apache/ozone/blob/e6b5d2dd12b4fe91bc141cf76bd17c170ed47f12/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteContainerCommandHandler.java#L96

nandakumar131 commented:
Thanks @adoroszlai for taking a look at this.
There were two more places which I missed
- CommandQueue
- TestReplicationManager

Fixed them all and updated the PR.

nandakumar131 commented:
Thanks @adoroszlai for the review and merge.
